api/server/router/local/image.go

```diff
@@ -484,7 +484,7 @@ func (s *router) postImagesTag(ctx context.Context, w http.ResponseWriter, r *ht
 	if err := s.daemon.TagImage(repo, tag, name, force); err != nil {
 		return err
 	}
-	s.daemon.EventsService.Log("tag", utils.ImageReference(repo, tag), "")
+	s.daemon.LogEvent("tag", utils.ImageReference(repo, tag), "")
```

Comment:
Let's wait for #17115 to be merged since that would remove this line and the need for daemon.LogEvent() entirely.